.changeset/polite-kids-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/hooks': patch
+---
+
+Adds `useMergeRefs` into `useMergeRef` directory
```

Comment:
No actual changes to the hook, right? Just moving the file & tests?

Reply:
no changes